python/docs/nc_direct.rst

```diff
@@ -0,0 +1,10 @@
+NcDirect documentation
```

Comment:
Do these have to be kept up-to-date by hand, or are they automatically generated?

Reply:
```
NcDirect documentation
======================

.. toctree::
    :maxdepth: 2
    :caption: Contents:

.. autoclass:: notcurses.NcDirect
    :members:
    :special-members: __init__
```

Automatic: new members of classes except the ones that start with double underscore.

New classes have to be added manually.